siliconcompiler/scheduler/schedulernode.py

```
@@ -365,18 +365,26 @@ def halt(self, msg: Optional[str] = None) -> None:
+            self.__record.set("status", NodeStatus.ERROR, step=self.__step, index=self.__index)
+            try:
+                self.__project.write_manifest(self.__manifests["output"])
+            except FileNotFoundError:
+                self.logger.error(f"Failed to write manifest for {self.__step}/{self.__index}.")
+
+            if errmsg:
+                self.logger.error(errmsg)
+            else:
+                self.logger.error(f"Halting {self.__step}/{self.__index} due to errors.")
+            send_messages.send(self.__project, "fail", self.__step, self.__index)
+        except:
```

Ruff (0.15.12)

[error] Do not use bare `except`

(E722)